Python/Product/ExternalProfilerDriver/VTuneInvoker.cs

```diff
@@ -0,0 +1,349 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Collections.Generic;
+using System.Text;
+
+using Microsoft.DotNet.PlatformAbstractions;
+
+namespace Microsoft.PythonTools.Profiling.ExternalProfilerDriver
+{
+    public class VTuneInvoker
+    {
+        private static readonly string _vtuneCl = @"\bin32\amplxe-cl.exe";
+        private static readonly string _vtune17Envvar = "VTUNE_AMPLIFIER_2017_DIR";
+        private static readonly string _vtune18Envvar = "VTUNE_AMPLIFIER_2018_DIR";
+
+        public static string VTunePath()
+        {
+            // expecting something like "C:\\Program Files (x86)\\IntelSWTools\\VTune Amplifier XE 2017";
+            string envvarval;
+            if (RuntimeEnvironment.OperatingSystemPlatform == Platform.Windows ||
+                 RuntimeEnvironment.OperatingSystemPlatform == Platform.Linux)
+            {
+                envvarval = Environment.GetEnvironmentVariable(_vtune17Envvar);
+                if (envvarval == null)
+                {
+                    envvarval = Environment.GetEnvironmentVariable(_vtune18Envvar);
+                }
+                if (envvarval == null)
+                {
+                    throw new VTuneNotInstalledException();
+                }
+
+            }
+            else
+            {
+                envvarval = "OS not supported"; // should this throw an exception?
+            }
+            if (File.Exists(envvarval + _vtuneCl)) // not exactly sure why Path.Combine doesn't work here
```

Comment:
`Path.Combine` should work, unless you had the leading backslash in `_vtuneCl` at the time. Without the backslash, it should be okay.

Also note that the backslashes won't work on Linux, and this line looks like it will be run in both.